docs/pages/guides/dom-components.mdx

```diff
@@ -62,26 +62,18 @@ export default function App() {
 
 To pass props to the underlying native **WebView**, add a `dom` object to the component:
 
-```tsx my-component.tsx (web)
-'use dom';
-
-export default function DOMComponent({}: { dom: import('expo/dom').DOMProps }) {
-  return (
-    <div>
-      <h1>Hello, world!</h1>
-    </div>
-  );
-}
-```
-
 Now you can pass [`WebView` props](https://github.com/react-native-webview/react-native-webview/blob/master/docs/Reference.md) to the DOM component:
 
 ```tsx App.tsx (native)
-import DOMComponent from './my-component';
+import { withDOMProps } from 'expo/dom';
+
+import MyComponent from './my-component';
+
+const MyComponentWithDOMProps = withDOMProps(MyComponent);
```

Comment:
```suggestion
import DOMComponent from './my-component';

const MyComponentWithDOMProps = withDOMProps(DOMComponent);
```

`MyComponent` should be `DOMComponent` here since that is exported from the code block below.